JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/query/RemoteStoreClient.java

```diff
@@ -350,14 +355,8 @@ public int executeUpdate(UpdateRequest update) {
      */
     @Override
     public int executeUpdate(String query) {
```

Comment:
It would be helpful to clean up the 3 executeUpdate methods a bit.
executeUpdate() does getConnectionUrl(), but this is then repeated in connect() when executeUpdate(String query) is called. Also some checks are performed on the url which are not performed in the other methods.